README.md

```diff
@@ -59,11 +61,20 @@ module.exports = function(config){
 }
 ```
 
-As you can see, the config file is *not* a simple JSON file, it should be a common js (a.k.a. npm) module.  
-You might recognize this way of working from the karma test runner.
+As you can see, the config file is *not* a simple JSON file. It should be a common js (a.k.a. npm) module. You might recognize this way of working from the karma test runner.
```

Comment:
It's actually not called an 'npm' module, but a 'node' module. :) just realize that now.